srcpkgs/Thunar/template

```diff
@@ -1,8 +1,7 @@
 # Template file for 'Thunar'
 pkgname=Thunar
-reverts="4.17.0_1"
```

Comment:
Please don't remove the `reverts` until `4.18.0`

Reply:
Re-added that line and force pushed the change.